Expose specific device_class for Velux covers

gibman opened the pull request:
Blinds, Rollingshutters and Awnings did not set their respective device_class attribute
Previously they would all appear as device_class "window"

## Breaking Change:

<!-- What is breaking and why we have to break it. Remove this section only if it was NOT a breaking change. -->

## Description:


**Related issue (if applicable):** fixes #<home-assistant issue number goes here>

**Pull request with documentation for [home-assistant.io](https://github.com/home-assistant/home-assistant.io) (if applicable):** home-assistant/home-assistant.io#<home-assistant.io PR number goes here>

## Example entry for `configuration.yaml` (if applicable):
```yaml

```

## Checklist:
  - [x] The code change is tested and works locally.
  - [x] Local tests pass with `tox`. **Your PR cannot be merged unless tests pass**
  - [x] There is no commented out code in this PR.
  - [x] I have followed the [development checklist][dev-checklist]

If user exposed functionality or configuration variables are added/changed:
  - [ ] Documentation added/updated in [home-assistant.io](https://github.com/home-assistant/home-assistant.io)

If the code communicates with devices, web services, or third-party tools:
  - [ ] [_The manifest file_][manifest-docs] has all fields filled out correctly. Update and include derived files by running `python3 -m script.hassfest`.
  - [ ] New or updated dependencies have been added to `requirements_all.txt` by running `python3 -m script.gen_requirements_all`.
  - [ ] Untested files have been added to `.coveragerc`.

If the code does not interact with devices:
  - [ ] Tests have been added to verify that the new code works.

[dev-checklist]: https://developers.home-assistant.io/docs/en/development_checklist.html
[manifest-docs]: https://developers.home-assistant.io/docs/en/creating_integration_manifest.html

gibman committed:
Update cover.py

Blinds, Rollingshutters and Awnings did not set their respective device_class attribute
Previously they would all appear as device_class "window"

OttoWinter commented on homeassistant/components/velux/cover.py:
```diff
@@ -60,8 +60,16 @@ def current_cover_position(self):
+        if isinstance(self.node, RollerShutter):
+            return 'shutter'
+        if isinstance(self.node, Awning):
+            return 'awning'
```
I think this should have an explicit `return None` at the end in case none of the if cases match

gibman committed:
fallback device class is always 'window'

fallback device class is always 'window' in the event we have an unknown cover type

Julius2342 commented:
The pipeline is blocked bc of two trivial linter errors:

```
homeassistant/components/velux/cover.py:63:80: E501 line too long (81 > 79 characters)
homeassistant/components/velux/cover.py:63:82: W291 trailing whitespace
```

@gibman : may you change them?

gibman committed:
trailing whitespace removed, trimmed as well

balloob committed:
Update cover.py
